mro/__init__.py

```diff
@@ -35,96 +35,120 @@ def execute_sql(sql, values=None):
 
 
 def _load_standard_db(connection):
+    print('Loading standard db')
     cursor = connection.cursor()
 
     tables = {}
 
     # Create any custom types
+    print('Creating custom types')
     mro.custom_types.create_custom_types(connection)
 
     # Get tables
+    print('Getting tables')
     cursor.execute("select * from information_schema.tables where table_schema='public';")
     connection.commit()
 
     for table in cursor:
         table_name = table[2]
+        print(f'Getting info about table [{table_name}]')
 
```

Comment:
In general, the use of the `information_schema` tables are more widely usable on other types of database (i.e. not Postgres) but are much, much slower than using the direct Postgres equivalents. Agreed with Andy that we can replace the generic implementation with Postgres-specific ones for now, and later on can refactor this into an appropriate implementation per database type if we decide to support other types of database.